From 9f53ef966f9bca7d53b364745f2495560139b30d Mon Sep 17 00:00:00 2001
From: Stanislav Levin <slev@altlinux.org>
Date: Wed, 27 Mar 2019 12:44:01 +0300
Subject: [PATCH] Make build configured against "with-gssapi" fail on missing
 GSSAPI

Bind just disables gssapi support if no GSSAPI/KRB5 headers found,
even if I say do it ("yes"). To prevent loss of GSSAPI support the
build should fail in such a case.

Fixes: https://gitlab.isc.org/isc-projects/bind9/issues/962
Signed-off-by: Stanislav Levin <slev@altlinux.org>
---
 bind/configure.ac | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/bind/configure.ac b/bind/configure.ac
index db5129a6ce..e2de928aa6 100644
--- a/bind/configure.ac
+++ b/bind/configure.ac
@@ -1024,12 +1024,15 @@ then
 				break
 			fi
 		fi
-		use_gssapi="no"
+		use_gssapi="not_found"
 	    done
 	fi
 fi
 
 case "$use_gssapi" in
+        not_found)
+                AC_MSG_ERROR([krb5.h or/and gssapi.h not found])
+                ;;
 	no)
 		AC_MSG_RESULT(disabled)
 		USE_GSSAPI=''
-- 
2.19.2

